### PR TITLE
changed navigation styles

### PR DESCRIPTION
**Pull request by mariuszmmm**
navigation styles changed to more responsive 😎

### CODE REVIEW

File: src/components/Navigation/styled.js [hunk 1]
```diff
@@ -23,64 +28,81 @@ export const Container = styled.nav`
...
   line-height: 40px;
+  letter-spacing: -1.5px;
+  text-align: left;
+  padding-right: 80px ;
+  min-width: max-content;
   transition: 0.6s;
+  
+  @media (max-width: ${({ theme }) => theme.breakpoint.medium}) {
+    padding-right: 19px ;
+    font-size: clamp(13px, 3vw, 13px);
```
Comment (Adrode) [inline src/components/Navigation/styled.js]: I would remove this line of code. With this font reduction, the app name appears very small on screens between 480px and 767px. In the first photo below there is a difference when this line of code is present and in the second one when we remove it.
1.


2.

**Review by Adrode — APPROVED** [review]
Nice work, really good job! Only one thing to correct, but the choice is up to you.

File: src/components/Navigation/styled.js (hunk 1 above)
Comment (mariuszmmm) [reply src/components/Navigation/styled.js]: Removed unnecessary font-size from AppName. 
It really is much better now.

**Review by Urbork — APPROVED** [review]
Great, now it looks even better

**Pull request merged**